tests/failingTests.js

```diff
@@ -22,7 +22,8 @@ window.failingTests = {
       'alias to id with same name',
       'ChangedExpressionSignal',
       'StringConversion',
-      'undefined property has undefined value'
+      'undefined property has undefined value',
+      'alias propagates it\'s changed signal back to referenced property'
```

Comment:
Could you insert this under the `'alias to id with same name',` instead?
I would reduce the affected lines count and would keep the list sorted.